src/main.rs

```diff
@@ -90,6 +90,10 @@ struct Opt {
     /// Ignore invalid watch pattern Regex on Applications. If flag is unset and an invalid Regex is found, the tool will exit with an error.
     #[structopt(long)]
     ignore_invalid_watch_pattern: bool,
+
+    // cluster name (only applicable to kind). Default: argocd-diff-preview
```

Comment:
```suggestion
    /// Cluster name (only applicable to kind)
```
1) `structopt` needs 3 `/` for rendering the `--help` output.
2) Since you are specifying `default_value = "argocd-diff-preview"`, you do not need to add the `Default: argocd-diff-preview` to the comment - `structopt` will add it automatically.

Reply:
ah i see, i am not familiar with rust or rust tooling